python/ray/_private/node.py

```diff
@@ -415,6 +415,38 @@ def _init_temp(self):
         )
         try_to_create_directory(self._runtime_env_dir)
 
+    def _get_node_labels(self):
+        def merge_labels(env_dict, params_dict):
+            """Merges two dictionaries, picking from the
+            first in the event of a conflict. Also emit a warning on every
+            conflict.
+            """
+
+            result = params_dict.copy()
+            result.update(env_dict)
+
+            for key in set(env_dict.keys()).intersection(set(params_dict.keys())):
+                if params_dict[key] != env_dict[key]:
+                    logger.warning(
+                        "Autoscaler is overriding your label:"
+                        f"{key}: {params_dict[key]} with {env_dict[key]}."
+                    )
+            return result
+
+        env_labels = {}
+        env_string = os.getenv(ray_constants.LABELS_ENVIRONMENT_VARIABLE)
+        if env_string:
+            try:
+                env_labels = json.loads(env_string)
+            except Exception:
+                logger.exception(f"Failed to load {env_string}")
+                raise
+            logger.debug(f"Autoscaler overriding labels: {env_labels}.")
```

Comment:
```suggestion
            logger.info(f"Autoscaler overriding labels: {env_labels}.")
```
Seems it is worth info?